CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Contributing to Red, Green, Refactor
```

Comment:
We could move this `CONTRIBUTING.md` file into the `.github` directory and only the most enterprising students would ever find it, but it would still show up on GitHub under contribution guidelines before making a PR.